.changeset/stupid-windows-press.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontity/wp-comments": patch
```

Comment:
It's technically a breaking change but it doesn't matter because we're in beta. Also the `errors` property is always an empty object so it should not be used by anybody for anything.